Sources/Markdown/Markdown.docc/Markdown/BlockMarkup.md

```diff
@@ -26,3 +26,5 @@
 
 ## See Also
 - <doc:BlockDirectives> 
+
+<!-- Copyright (c) 2021-2022 Apple Inc and the Swift Project authors. All Rights Reserved. -->
```

Comment:
I feel like these missing license headers should get their own PR that can be reviewed and merged quickly - my own outstanding PRs here also have commits that add these 😅